ddtrace/_monkey.py

```diff
@@ -200,7 +203,11 @@ def patch_iast(**patch_modules):
     """
     iast_enabled = formats.asbool(os.environ.get(IAST_ENV, "false"))
     if iast_enabled:
-        patch(raise_errors=False, patch_modules_prefix="ddtrace.appsec.iast.taint_sinks", **patch_modules)
+        # TODO: Devise the correct patching strategy for IAST
+        for module in (m for m, e in patch_modules.items() if e):
+            when_imported("hashlib")(
```

Comment:
Why `hashlib` here? This would benefit from clarification in the docstring.

Reply:
This is just the quickest way to make the patching happen on import. I've added a comment there to remind myself to also check with ASM if this would work with them, or if we should have a better pattern for this sort of things.